e2e/vm/vm_test.go

```diff
@@ -34,6 +36,11 @@ func TestVM(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	limactlO, limaHomePath, err := e2e.CreateLimaOption()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
let's move this into `soci_test.go` as a BeforeEach (https://onsi.github.io/ginkgo/#extracting-common-setup-beforeeach). then you can simplify the `testSoci` function signature, too